### PR TITLE
[Mailer][Notifier] Add webhooks signature verification on Sweego bridges

### PR DESCRIPTION
**Pull request by welcoMattic**
| Q             | A
| ------------- | ---
| Branch?       | 7.3
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no 
| Issues        | Fix #... 
| License       | MIT

This PR follows the 2 previous ones (https://github.com/symfony/symfony/pull/57431, and https://github.com/symfony/symfony/pull/58322). It brings webhook signature verification to improve security.

Sweego documentation about webhook is here https://learn.sweego.io/docs/webhooks/webhook_signature

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

**Review by stof — COMMENTED** [review]
Can you add tests covering this logic ?

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
@welcoMattic In addition to adding new tests, can you have a look at the existing ones that are broken now.

**Comment by fabpot** [comment]
@welcoMattic Last chance to include this PR in 7.2.

**Comment by welcoMattic** [comment]
I couldn't make it for 7.2, sorry. Let's remove it from FF. 

(I'm on my honeymoon trip until 28th nov.)

**Comment by fabpot** [comment]
> (I'm on my honeymoon trip until 28th nov.)

Enjoy!

**Comment by welcoMattic** [comment]
I've added tests and fix existing ones. Failure is related to something on HttpClient itself, not related to this PR.

File: src/Symfony/Component/Notifier/Bridge/Sweego/composer.json [hunk 1]
```diff
@@ -23,6 +23,9 @@
     "require-dev": {
         "symfony/webhook": "^6.4|^7.0"
     },
+    "conflict": {
+        "symfony/http-foundation": "<7.1"
```
Comment (Nyholm) [inline src/Symfony/Component/Notifier/Bridge/Sweego/composer.json:27]: How come you need this?

**Review by Nyholm — APPROVED** [review]
Thank you. Just one comment

File: src/Symfony/Component/Notifier/Bridge/Sweego/composer.json (hunk 1 above)
Comment (Nyholm) [reply src/Symfony/Component/Notifier/Bridge/Sweego/composer.json:27]: Because of webhook rejection exception

**Comment by fabpot** [comment]
Thank you @welcoMattic.

**Pull request merged**